src/v3/src/transformer/terminal/transformTerminalMessages.ts

```diff
@@ -143,6 +144,12 @@ export const transformTerminalMessages: TerminalKeyTransformer = (transaction, f
   } else if (containsMessageKeyPrefix('core.auth.factor.signedNonce.error', displayedMessages)) {
     // custom title for signed nonce errors
     displayedMessages[0].title = loc('user.fail.verifyIdentity', 'login');
+  } else if (containsMessageKeyPrefix('idx.error.code.access_denied.device_assurance.remediation', displayedMessages)) {
+    // OKTA-630044 - messages from rawIdxState are used until this issue is solved
+    uischema.elements.push(
+      buildEndUserRemediationError(transaction.rawIdxState.messages?.value || []),
+    );
+    return formBag;
```

Comment:
```suggestion
    const userRemediationErrorElement = buildEndUserRemediationError(transaction.rawIdxState.messages?.value || []);
    if (userRemediationErrorElement) {
      uischema.elements.push(userRemediationErrorElement);
    }
    return formBag;
```

This is just in case there is an API issue that doesn't return the expected messages array, we will not display an empty error box.